client/src/controllers/index.ts

```diff
@@ -25,6 +26,10 @@ export const coreControllerDefinitions: Definition[] = [
   { controllerConstructor: CountController, identifier: 'w-count' },
   { controllerConstructor: DismissibleController, identifier: 'w-dismissible' },
   { controllerConstructor: MessagesController, identifier: 'w-messages' },
+  {
+    controllerConstructor: PreviewPanelController,
+    identifier: 'w-preview-panel',
```

Comment:
small aside, using a single word makes the data attributes overall less verbose.

Maybe `w-preview`?

Reply:
I was tempted but I wasn't sure because we have other ways to access previews. That said, I think the panel has become the primary way to use previews, and since the code also controls the "preview in new tab" functionality, I think it makes sense to name this `PreviewController`. Updated.

Reply:
Epic